src/main/java/org/opensearch/security/dlic/rest/api/RestApiAdminPrivilegesEvaluator.java

```diff
@@ -66,6 +66,7 @@ default String build() {
         .put(Endpoint.CONFIG, action -> buildEndpointActionPermission(Endpoint.CONFIG, action))
         .put(Endpoint.INTERNALUSERS, action -> buildEndpointPermission(Endpoint.INTERNALUSERS))
         .put(Endpoint.NODESDN, action -> buildEndpointPermission(Endpoint.NODESDN))
+        .put(Endpoint.RATELIMITERS, action -> buildEndpointPermission(Endpoint.RATELIMITERS))
```

Comment:
Will this also need updated in [AbstractApiValidationTest](https://github.com/opensearch-project/security/blob/main/src/test/java/org/opensearch/security/dlic/rest/api/AbstractApiActionValidationTest.java#L138) and [EndpointValidatorTest](https://github.com/opensearch-project/security/blob/main/src/test/java/org/opensearch/security/dlic/rest/validation/EndpointValidatorTest.java)?